### DISCUSSION TITLE
Allure Testops cloud is compatible with bitbucket pipeline?

### DISCUSSION THREAD

**Discussion by xdsanzxd** [allure-framework/allure2#1842] [TestOps]
I mean, exist the possibility to send the reports from wdio project to allure testops cloud?

Because I was searched and the integration with bitbucket CI is not done yet >
https://docs.qameta.io/allure-testops/integrations/bitbucket/main/
I'm looking a way to work with bitbucket pipeline + WDIO + allure testops cloud
someone knows something about it?
Thanks in advance

**Comment by cheshi-mantu** [comment]
@xdsanzxd Hi Fernando,
we might have a solution for you, please come to https://help.qameta.io (which is a proper way to get Allure TestOps tech support).
Please use your work email address for the registration and help desk portal.
cheers

**Reply by cheshi-mantu** [reply]
@xdsanzxd Hi Fernando
We've added Bitbucket cloud as CI integration in Allure TestOps and allurectl.

<img width="637" alt="image" src="https://user-images.githubusercontent.com/8607348/212738466-709e7a45-4ed0-4ede-b69e-893e4f41ad9f.png">
